src/AdvancedMH.jl

```diff
@@ -51,14 +51,14 @@ logdensity(model::DensityModel, t::Transition) = t.lp
 
 # A basic chains constructor that works with the Transition struct we defined.
 function AbstractMCMC.bundle_samples(
-    ts::Vector{<:Transition},
+    ts::Vector{Transition{T, L}},
```

Comment:
```suggestion
    ts::Vector{<:Transition},
```

Reply:
I tried that, but the compiler can't dispatch on it correctly -- it always fall into this method.

Reply:
Wait, nevermind, it works if you don't use the typing nonesense I had below.